### PR TITLE
Register timmy.is-a.dev

### PR DESCRIPTION
**Pull request by BunkDev5674**
Register timmy.is-a.dev with A record pointing to 188.93.139.120.

### CODE REVIEW

File: domains/timmy.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "BunkDev5674",
+           "email": "79503178+BunkDev5674@users.noreply.github.com",
```
Comment (DEV-DIBSTER) [inline domains/timmy.json:4]: Cannot use a no reply email address.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/timmy.json (hunk 1 above)
Comment (BunkDev5674) [reply domains/timmy.json:4]: i edited it

File: domains/timmy.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "BunkDev5674",
+           "email": "bunkdevelopers@gmail.com",
+           "discord": "845570884776230943"
+        },
+    
+        "record": {
+            "A": ["188.93.139.120"]
```
Comment (CuteDog5695) [inline domains/timmy.json:9]: 404 not found, please give a preview of your website

File: domains/timtim.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "BunkDev5674",
+           "email": "bunkdevelopers@gmail.com",
+           "discord": "845570884776230943"
+        },
+    
+        "record": {
+            "A": ["188.93.139.120"]
```
Comment (CuteDog5695) [inline domains/timtim.json:9]: 404 not found, please give a preview of your website

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**